### ISSUE TITLE
pip install fails with "undefined reference to `PyParser_SimpleParseString'" in Python 3.10rc2

### ISSUE THREAD

**Issue by bunchesofdonald** [unbit/uwsgi#2350]
When attempting to install uwsgi in python 3.10rc2 I'm getting a failure saying that `PyParser_SimpleParseString` and `PyNode_Compile` are undefined. 

```
docker run --entrypoint='/bin/bash' -it python:3.10.0rc2
root@d6912d2c670f:/# pip install uwsgi
Collecting uwsgi
  Downloading uWSGI-2.0.19.1.tar.gz (803 kB)
     |████████████████████████████████| 803 kB 1.9 MB/s
Building wheels for collected packages: uwsgi
  Building wheel for uwsgi (setup.py) ... error
  ERROR: Command errored out with exit status 1:
  ...
    pyloader.c:(.text+0x170f): undefined reference to `PyParser_SimpleParseString'
    /usr/bin/ld: pyloader.c:(.text+0x1728): undefined reference to `PyNode_Compile'
```

**Comment by xrmx** [comment]
@bunchesofdonald does it build master or the uwsgi-2.0 branch? Command to build from a git checkout is `python3 uwsgiconfig.py --build`

**Comment by bunchesofdonald** [comment]
That seems to do the trick. It installs and seems to startup as expected.

**Comment by xrmx** [comment]
@unbit we need a 2.0.20 please :)

**Comment by ericsouza** [comment]
same error here after I upgrade my python to `3.10.0`

**Comment by xrmx** [comment]
This should be fixed in 2.0.20
